### DISCUSSION TITLE
README.md有图片链接

### DISCUSSION THREAD

**Discussion by tkfanswind** [spencerwooo/onedrive-vercel-index#204] [Q&A]
Access to XMLHttpRequest at 'https://xxxx-my.sharepoint.com/personal/xxxx/_layouts/15/download.aspx?' from origin 'https://komuro.cf' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

README.md有图片链接报错。请问如何解决？

**Comment by spencerwooo** [comment]
CORS 问题，你这个 `https://xxxx-my.sharepoint.com/personal/xxxx/_layouts/15/download.aspx` 是什么图片有这样的链接？
